docs/discover/search-sessions.asciidoc

```diff
@@ -37,19 +36,24 @@ open your browser and pick up where you left off.
 
 . Load your dashboard.
 +
-Your search session begins automatically. The icon after the dashboard title
-displays the current state of the search session. A clock indicates the search session is in progress.
+Your search session begins automatically.
+The icon after the dashboard title displays the current state of the search session.
+A clock icon indicates the search session is in progress.
 A checkmark indicates that the search session is complete.
 
 . To instruct {kib} to continue a search in the background, click the clock icon,
```

Comment:
Super nit: Add the clock icon image?

Reply:
It's shown in the image in the next step.